lib/absinthe/schema.ex

```diff
@@ -613,7 +613,7 @@ defmodule Absinthe.Schema do
   def to_sdl(schema) do
     pipeline =
       schema
-      |> Absinthe.Pipeline.for_schema()
+      |> Absinthe.Pipeline.for_schema(prototype_schema: schema.__absinthe_prototype_schema__)
```

Comment:
This was needed to get the directive definitions in place when applied with a prototype schema